cloud/amazon/ec2_elb_lb.py

```diff
@@ -142,6 +143,21 @@
     required: false
     default: 60
     version_added: "2.1"
+  tags:
+    description: the tags dict to apply
+    type: dict
+    required: false
+    sample: { "Department": "Information Retrieval" }
+    version_added: "2.1"
+  purge_tags:
```

Comment:
"purge_tags" is able to be set True independent of "tags" being defined, so as to delete all existing tags from an ELB.  For this reason, "append_tags" may not be an accurate name?  It follows the pattern in ec2_group [purge_rules](https://github.com/ansible/ansible-modules-core/blob/devel/cloud/amazon/ec2_group.py#L56) .... I can certainly rename "purge_tags" to "append_tags", and make it require "tags" to be defined, but im not sure this is the desired functionality as it would lose the tag purging ability.  @defionscode @mmochan @smiller171 @wimnat @tombamford @jsdalton @rickmendes thoughts?